.travis.yml

```diff
@@ -97,6 +99,7 @@ install:
     export CI_RELEASE=true
   fi
 script:
+- hlint --git
```

Comment:
I think having the `ci/build.sh` script call out to hlint would be my preference. I suspect it doesn't really matter whether it's included in the timeout or not, because I don't suspect running hlint takes very long, but I think it's preferable to only have to look in one place to see what each CI job is actually checking.

Reply:
What does the `--git` flag do? I haven't managed to find documentation for it.

Reply:
Sounds good.

`--git` finds only the tracked files and lints them.